### DISCUSSION TITLE
@user_script: 36: Unknown Redis command called from Lua script

### DISCUSSION THREAD

**Discussion by remior** [netbox-community/netbox#10376] [Q&A]
hello

i got an error after creating superuser in netbox

<class 'redis.exceptions.ResponseError'>

Error running script (call to f_0605214935a9ffcd4b9e5779300302540ff08da4): @user_script:36: @user_script: 36: Unknown Redis command called from Lua script 

Python version: 3.8.12
NetBox version: 2.10.2

i had a look in discussion and people explain that i need to upgrade redis

and it is what i did : sh-4.2$ redis-server -v
Redis server v=6.2.4 sha=00000000:0 malloc=jemalloc-5.1.0 bits=64 build=3934c45355244da4

but i m still having the same issue

thanks for your help

**Comment by candlerb** [comment]
What does this command show?

```
redis-cli info | grep redis_version
```

**Comment by remior** [comment]
hum  something strange indeed
`sh-4.2$ redis-cli info | grep redis_version
redis_version:3.2.12
`

**Reply by candlerb** [reply]
The `info` command asks the server what version it is running.

It's possible you didn't restart the redis server after upgrading.  Or it's possible you now have two versions of redis-server installed in different locations (say, /usr/bin and /usr/local/bin). Then either systemd is just running the old version; or the old version is binding to port 6379, which prevents the new version binding to this port.

In any case, you know now that the problem is that redis-server *hasn't* been correctly upgraded, so you know where to investigate.

**Comment by remior** [comment]
indeed...

I found 2 :
/usr/bin/redis-server
/usr/local/bin/redis-server

thanks I ll check that and leave a comment here when solved

**Reply by copartk** [reply]
@remior 

Did you find the fix for the above issue?
